src/language/CSSUtils.js

```diff
@@ -83,19 +83,31 @@ define(function (require, exports, module) {
     
     /**
      * @private
+     * Returns state of a token
+     * @param {{editor:{CodeMirror}, pos:{ch:{string}, line:{number}}, token:{object}}} ctx
+     * @return {{tokenize:function, state:string, stateArg:string, context:Object}}
+     */
+    function _getState(ctx) {
+        var state = ctx.token.state.localState || ctx.token.state;
+        if (!state.context && ctx.token.state.html.localState) {
+            state = ctx.token.state.html.localState;
+        }
+        return state;
+    }
+
+    /**
+     * @private
      * Checks if the current cursor position is inside the property name context
      * @param {editor:{CodeMirror}, pos:{ch:{string}, line:{number}}, token:{object}} context
      * @return {boolean} true if the context is in property name
      */
     function _isInPropName(ctx) {
-        var state,
+        var state = _getState(ctx),
             lastToken;
-        if (!ctx || !ctx.token || !ctx.token.state || ctx.token.type === "comment") {
+        if (!state || !ctx || !ctx.token || ctx.token.type === "comment") {
             return false;
```

Comment:
Here, and twice below, you should move the check at the end of the `if` (after you are sure ctx is valid)
or simply move the assignment after the first `if`.